ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -195,11 +214,16 @@ export default class AmendRegistration extends Vue {
   }
 
   private confirmAmendment (): void {
-    this.$router.push({
-      name: RouteNames.CONFIRM_DISCHARGE,
-      query: { 'reg-num': this.registrationNumber }
-    })
-    this.emitHaveData(false)
+    if (this.debtorValid && this.securedPartiesValid && this.registrationLengthTrustValid && this.collateralValid) {
+      // TODO: change route name
+      this.$router.push({
+        name: RouteNames.CONFIRM_DISCHARGE,
+        query: { 'reg-num': this.registrationNumber }
+      })
+      this.emitHaveData(false)
+    } else {
+      this.showDebtorInvalid = true
```

Comment:
Hey steph, could this variable just be 'showInvalid' so we can use it for all the editable components later?

Reply:
done